lightning/src/offers/invoice.rs

```diff
@@ -761,13 +771,36 @@ macro_rules! invoice_accessors { ($self: ident, $contents: expr) => {
 	}
 } }
 
+macro_rules! invoice_accessors_signing_pubkey {
+	($self: ident, $contents: expr, $invoice_type: ty) =>
+{
+	/// The public key corresponding to the key used to sign the invoice.
+	///
+	/// If the invoices was created in response to an [`Offer`], then this will be:
+	/// - [`Offer::issuer_signing_pubkey`] if it's `Some`, otherwise
+	/// - the final blinded node id from a [`BlindedMessagePath`] in [`Offer::paths`] if `None`.
+	///
+	/// If the invoice was created in response to a [`Refund`], then this may be a transient id
```

Comment:
Updated as:

```diff
diff --git a/lightning/src/offers/invoice.rs b/lightning/src/offers/invoice.rs
index 551f36510..eae1c14dc 100644
--- a/lightning/src/offers/invoice.rs
+++ b/lightning/src/offers/invoice.rs
@@ -775,14 +775,14 @@ macro_rules! invoice_accessors { ($self: ident, $contents: expr) => {
 macro_rules! invoice_accessors_signing_pubkey {
        ($self: ident, $contents: expr, $invoice_type: ty) =>
 {
-       /// The public key corresponding to the key used to sign the invoice.
+       /// A typically transient public key corresponding to the key used to sign the invoice.
        ///
        /// If the invoices was created in response to an [`Offer`], then this will be:
        /// - [`Offer::issuer_signing_pubkey`] if it's `Some`, otherwise
        /// - the final blinded node id from a [`BlindedMessagePath`] in [`Offer::paths`] if `None`.
        ///
-       /// If the invoice was created in response to a [`Refund`], then this may be a transient id
-       /// chosen by the recipient.
+       /// If the invoice was created in response to a [`Refund`], then it is a valid pubkey chosen by
+       /// the recipient.
        ///
        /// [`Offer`]: crate::offers::offer::Offer
        /// [`Offer::issuer_signing_pubkey`]: crate::offers::offer::Offer::issuer_signing_pubkey
```